src/main/java/github/scarsz/discordsrv/util/WebhookUtil.java

```diff
@@ -113,70 +119,54 @@ public static void deliverMessage(TextChannel channel, Player player, String mes
         });
     }
 
-    static class LastWebhookInfo {
-
-        final String webhook;
-        final String targetName;
+    private static final Map<String, List<String>> channelWebhookIds = new ConcurrentHashMap<>();
+    private static final Map<String, String> lastUsedWebhookIds = new ConcurrentHashMap<>();
+    private static final int webhookPoolSize = 2;
```

Comment:
Did you want this exposed in config.yml?

Reply:
2 is fine, it was initially 2 because of Android (even though that didn't fix the issue), but it's only use would be allowing more bandwidth, iirc the rate limit for webhooks is 5/5, so that would make it 10/5 so 1 message every half a second (iirc)